index.js

```diff
@@ -31,7 +32,7 @@ Sentry.init({
   // We recommend adjusting this value in production.
   environment: getEnvironmentName(),
   tracesSampleRate: 1.0,
-  release: '1.11.6',
+  release: getReadableVersion(),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getReadableVersion()` function is used to set the release version for Sentry. This could potentially lead to issues if the function doesn't return a valid version string. It would be better to add error handling around this to ensure that a valid version string is always provided.

```diff
-  release: getReadableVersion(),
+  release: getReadableVersion() || 'unknown',
```
This will ensure that even if `getReadableVersion()` fails for some reason, we still have a default value for the release.


<!-- This is an auto-generated comment by OSS CodeRabbit -->